pxf/pxf-service/src/scripts/pxf-service

```diff
@@ -22,8 +22,8 @@
 
 pxf_root=/usr/lib/pxf
 env_script=/etc/pxf/conf/pxf-env.sh
-tomcat_root=/opt/pivotal/apache-tomcat
-tomcat_templates=/opt/pivotal/pxf/tomcat-templates
+tomcat_root=/opt/apache-tomcat
+tomcat_templates=/opt/pxf/tomcat-templates
```

Comment:
Same holds true even for the library and conf paths as well being referenced in the classpath file (using their symlinks). Should be fine, as the expectation is that pxf would be installed using the virtual rpm